packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -97,7 +97,6 @@ const { floatingStyles, middlewareData } = useFloating(targetRef, floatingRef, {
     <slot />
   </div>
   <ScalarTeleport
-    v-if="$slots.floating"
```

Comment:
This is important iirc. I think there are teleport issues if we try to create this without the floating element 🤔

Reply:
The combobox ones I think should be removed too otherwise empty divs get included.

Reply:
We need to validate this behaviour with latest Vue. I am not sure if it's vue-tsc or a change to vue internals but TS is saying `$slots.floating` is always defined.